frontend/webapp/views/main/main_view.py

```diff
@@ -57,27 +57,18 @@ def display_hijack():
     _mitigation_flag = False
 
     hijack_data = get_hijack_by_key(_key)
-    _configured = False
 
     if hijack_data is None:
         app.artemis_logger.debug("Hijack with id found: {}".format(_key))
         return render_template("404.htm")
 
-    if "configured_prefix" in hijack_data:
```

Comment:
this is removed so that we do not have to check in a broken-down prefix list

Reply:
@pgigis please check if removing this flag is ok. I think that if the prefix is not configured, the hijack event will become automatically outdated!